charts/pulsar/templates/broker/function-worker-configfile-configmap.yaml

```diff
@@ -84,6 +84,19 @@ data:
       changeConfigMapNamespace: {{ template "pulsar.namespace" . }}
       submittingInsidePod: true
       installUserCodeDependencies: true
+  {{- if .Values.functions.functionMesh }}
+  functionsWorkerServiceNarPackage: "/pulsar/lib/mesh-worker-service.nar"
+  functionsWorkerServiceCustomConfigs:
+    extraDependenciesDir: "/pulsar/lib"
```

Comment:
should add `jobNamespace` to `functionsWorkerServiceCustomConfigs`, since we already depreacted the config from `functionRuntimeFactoryConfigs`.